spec/system/page_objects/modals/flag.rb

```diff
@@ -30,7 +30,7 @@ def check_confirmation
       end
 
       def has_choices?(*choices)
-        body.all(".flag-action-type-details strong").map(&:text) == choices
+        expect(body.all(".flag-action-type-details strong").map(&:text)).to eq(choices)
```

Comment:
This makes it easier to see what is different when the test fails instead of just getting a boolean result.